compaction/compaction_manager.cc

```diff
@@ -1771,8 +1771,6 @@ void compaction_backlog_tracker::transfer_ongoing_charges(compaction_backlog_tra
             new_bt.register_compacting_sstable(w.first, *w.second);
         }
     }
-    _ongoing_writes = {};
-    _ongoing_compactions = {};
```

Comment:
So it will temporarily double the backlog? I see no problem with it, but I hope there's no problem with an sstable being registered twice.

Reply:
we don't yield until the end of procedure, and when it happens, only one tracker will be actually registered.

Reply:
Anyway, by the end of the series, the new backlog tracker is only registered in the execution step.